docs/scripting-element-object.rst

```diff
@@ -0,0 +1,395 @@
+.. _splash-element:
+
+Element Object
+==============
+
+Element objects are instanced by :ref:`splash-select`;
+
+.. _splash-element-attributes:
+
+Attributes
+~~~~~~~~~~
+
+The following fields are read-only.
+
+.. _splash-element-id:
+
+element.id
+----------
+
+Id of the inner representation of the element.
+
+Methods
+~~~~~~~
+
+To modify or retrieve some information about the element you can use the following methods.
+Note that all of the following methods return flag indicating whether the operation was
+successful or not. The methods returns ``ok, reason_or_value`` pair. If ``ok`` is nil
+then error happened during the operation; ``reason`` provides an information about error type;
+otherwise ``ok`` is ``true`` and the returned value is stored in the second variable.
+
+
+.. _splash-element-exists:
+
+element:exists
+--------------
+
+Check whether the element exists in DOM. If the element doesn't exist almost all other
+methods will fail.
+
+**Signature:** ``ok, exists = element:exists()``
+
+**Returns:** ``ok, exists`` pair. If ``ok`` is nil then error happened during the check;
+``exists`` provides an information about error type; otherwise ``exists`` indicated whether
+the element exists.
```

Comment:
You are right. Particularly this method has very little chance to fail (or maybe doesn't have at all).